pkg/registry/apis/provisioning/test.go

```diff
@@ -167,6 +168,12 @@ func ValidateRepository(repo repository.Repository) field.ErrorList {
 			"The target type is required when sync is enabled"))
 	}
 
+	resyncIntervalSeconds := int64(ResyncInterval / time.Second)
+	if cfg.Spec.Sync.Enabled && cfg.Spec.Sync.IntervalSeconds < resyncIntervalSeconds {
```

Comment:
it can be zero -- and not run

Reply:
no the case. It defaults to 10 second.

Reply:
can we change that?  it seems better to explicitly set a sync interval.  For many (most?!) setups will only want to look once at startup.